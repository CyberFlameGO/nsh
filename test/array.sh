arr=(@ this is "Hello World" is message @@)
echo -n $arr
echo ${arr[0]}
echo ${arr[1]}
echo ${arr[1 + 1]}
echo ${arr[3]}
echo ${arr[4]}
echo ${arr[5]}
echo ${arr[2 * 3]}
