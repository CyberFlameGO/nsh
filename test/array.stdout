@@
this
is
Hello World
is
message
@@
